bin/prettier.js

```diff
@@ -171,17 +171,15 @@ if (stdin) {
 } else {
   eachFilename(filepatterns, filename => {
     fs.readFile(filename, "utf8", (err, input) => {
-      if (write) {
-        console.log(filename);
-      }
-
       if (err) {
```

Comment:
So we need to log twice? Cause here we don't know yet - should we change file or not.